### PR TITLE
Improve contributor guidelines

### PR DESCRIPTION
**Pull request by adambkaplan**
* Move existing contributing guide to docs/HACKING.md
* Created new CONTRIBUTING.md focusing on submitting issues and PRs.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,113 +1,127 @@
...
+Any member of the OpenShift organization can review your PR and signal their approval with the
+`/lgtm` command. Only approvers listed in the [OWNERS](OWNERS) file may add the `approved` label to
+your PR. In order for PR to merge, it must have the following:
 
+1. The `approved` label
+2. The `lgtm` label
+3. All tests passing in CI, which is managed by Prow.
 
-### Integration tests
+If you are no a member of the OpenShift GitHub organization, an OpenShift team member will need to
```
Comment (bparees) [inline CONTRIBUTING.md]:
```suggestion
If you are not a member of the OpenShift GitHub organization, an OpenShift team member will need to
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,113 +1,127 @@
...
+to a GitHub issue, add it here with the text `Fixes xxx`.
 
-## Building a Release
+### Work In Progress PRs
 
-To build a S2I release you run `make release` on a system with Docker,
-which will create a build environment image and then execute a cross platform Go build within it. The build
-output will be copied to `_output/releases` as a set of tars containing each version.
+Contributors who would like feedback but are still making code modifications can create "work in
+progress" PRs by adding "WIP" to the PR title. Once work is done and ready for final reivew, please
```
Comment (bparees) [inline CONTRIBUTING.md]:
```suggestion
progress" PRs by adding "WIP" to the PR title. Once work is done and ready for final review, please
```

**Comment by adambkaplan** [comment]
@bparees fixed & squashed

**Pull request merged**